practices/smart_goals.md

```diff
@@ -24,6 +24,8 @@ SMART Goals is a tool that can be used to plan and achieve your goals – both p
 
 ![](/files/smart3.png) 
 
+[SMART goals summary *(author)*](smart3.png)
```

Comment:
invalid image link

Reply:
source should navigate to any website - if it is your image, please navigate e.g. to dxheroes.io

Reply:
Link changed.